satpy/multiscene.py

```diff
@@ -293,6 +293,18 @@ def resample(self, destination=None, **kwargs):
     def blend(self, blend_function=stack):
         """Blend the datasets into one scene.
 
+        Reduce the ``MultiScene`` to a single ``Scene``.  Datasets
+        occurring in each scene will be passed to a blending
+        function, which shall take as input a list of datasets
+        (:class:`xarray.DataArray` objects) and shall return a single
+        dataset (:class:`xarray.DataArray` object).  The blend method
+        then assigns those datasets to the blended scene.
+
+        Blunding functions provided in this module are :func:`stack`
+        (the default) and :func:`timeseries`, but the Python built-in
+        function function `sum` also works and may be appropriate for
+        some types of data.
```

Comment:
```suggestion
        Blending functions provided in this module are :func:`stack`
        (the default) and :func:`timeseries`, but the Python built-in
        function `sum` also works and may be appropriate for
        some types of data.
```

Do these `func` links render properly when the website is generated?

Reply:
They do now!